docs/examples/rbac/nodejs/Makefile

```diff
@@ -5,4 +5,6 @@ test: node_modules
 
 node_modules: package.json
 	yarn
+	cd ../../../../languages/js; yarn link
```

Comment:
Is this going to leave us in `../../../../languages/js` for the following steps in this task or will we pop back to `docs/examples/rbac/nodejs`?

Reply:
No, it'll bring you back. TIL a number of things about making, including that it runs each command as a subprocess, so your `cd`s only affect that one line.